examples/geo_schelling_points/geo_schelling_points/agent.py

```diff
@@ -0,0 +1,75 @@
+import random
```

Comment:
This filename needs to be agents.py

Reply:
Updated filename. Changed `geo_sir/agent.py` to `geo_sir/agents.py` too.